ci/build/build-code-server.sh

```diff
@@ -32,14 +31,10 @@ main() {
     set -e
   fi
 
-  parcel build \
-    --public-url "." \
-    --dist-dir dist \
-    $([[ $MINIFY ]] || echo --no-optimize) \
-    src/browser/register.ts \
-    src/browser/serviceWorker.ts \
-    src/browser/pages/login.ts \
-    src/browser/pages/vscode.ts
+  yarn browserify src/browser/register.ts -o out/browser/register.browserified.js
+  yarn browserify src/browser/serviceWorker.ts -o out/browser/serviceWorker.browserified.js
+  yarn browserify src/browser/pages/login.ts -o out/browser/pages/login.browserified.js
+  yarn browserify src/browser/pages/vscode.ts -o out/browser/pages/vscode.browserified.js
```

Comment:
Before, we just did `parcel`. Do you need to use `yarn` or can we call the CLI directly?

Reply:
I think we can just use `browserify` because `yarn` adds the bin directory to the path but that assumes no one calls the script directly so it seems safer this way.

Reply:
Ah, thanks for explaining that! Makes sense. Okay, interpreting that as "leave as is"!